### PR TITLE
fix a LateInitializationError

### PR DESCRIPTION
**Pull request by xu-baolin**
fixes https://github.com/flutter/flutter/issues/83618

### CODE REVIEW

File: lib/web_ui/lib/src/engine/text_editing/text_editing.dart [hunk 1]
```diff
@@ -1595,6 +1603,7 @@ class TextEditingChannel {
         // There's no API to request autofill on the web. Instead we let the
         // browser show autofill options automatically, if available. We
         // therefore simply ignore this message.
+        command = const TextInputRequestAutofill();
```
Comment (LongCatIsLooong) [inline lib/web_ui/lib/src/engine/text_editing/text_editing.dart:1606]: Maybe remove the `late` on line 1555?

Comment (LongCatIsLooong) [reply lib/web_ui/lib/src/engine/text_editing/text_editing.dart:1606]: according to the comments I think it's safe to just return here.

Comment (xu-baolin) [reply lib/web_ui/lib/src/engine/text_editing/text_editing.dart:1606]: Just keep it consistent with other unsupported commands, such as 'TextInputSetCaretRect'.

**Comment by LongCatIsLooong** [comment]
RSLGTM for autofill. Can we remove the `late` keyword so it becomes a compile-time error?

File: lib/web_ui/test/text_editing_test.dart [hunk 2]
```diff
@@ -435,6 +435,13 @@ void testMain() {
       textEditing = null;
     });
 
+    test('TextInput.requestAutofill', () async {
+      final MethodCall requestAutofill = MethodCall('TextInput.requestAutofill');
+      sendFrameworkMessage(codec.encodeMethodCall(requestAutofill));
+
+      //No-op and without crashing.
```
Comment (yjbanov) [inline lib/web_ui/test/text_editing_test.dart:442]: We could check that the `spy` intercepted a response, which would prove that the engine did process the message. For example:

```
      expect(spy.messages, hasLength(1));
```

**Review by yjbanov — APPROVED** [review]
LGTM with a small suggestion to improve the test.

**Comment by sarbogast** [comment]
When will this fix be available? Most of my forms crash and end up having several text fields focused at the same time, and I suspect it's because of this exception.

**Pull request merged**